src/Map/src/Renderer/AbstractRenderer.php

```diff
@@ -81,4 +81,21 @@ final public function renderMap(Map $map, array $attributes = []): string
 
         return \sprintf('<div %s></div>', $stimulusAttributes);
     }
+
+    private function getMapAttributes(Map $map): array
+    {
+        $computeId = fn (array $array) => hash('xxh3', json_encode($array));
```

Comment:
Not sure if a method is required here ?

Reply:
As you mentioned in https://github.com/symfony/ux/pull/2385/files#r1851075522, we may want to add more logic to compute the identifier, so I've laid the foundations for the future :)